gpii/node_modules/preferencesServer/src/preferencesService.js

```diff
@@ -73,6 +108,41 @@ var fluid = fluid || require("infusion");
                 listener: "gpii.preferencesServer.preferencesService.updateGpiiKey",
                 args: ["{dataStore}.updateGpiiKey", "{arguments}.0"],
                 namespace: "updateGpiiKey"
+            }],
+            onCloudSafeCredCreate: [{
+                func: "{dataStore}.findById",
```

Comment:
When prefs safe id is not found, the operation should stop with an error.

Reply:
This has been fixed.